### ISSUE TITLE
Device images are too tiny

### ISSUE THREAD

**Issue by xkrypt0nx** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#1704]
We can add images for each device, but the images show too small. I am creating a website and using the appointment widget, and on desktop or mobile view, the image is super small. Here is an example https://prnt.sc/1slfn8k

**Comment by xkrypt0nx** [comment]
It would look way better if it could fill in the squares better

**Comment by Repairdesk-QA** [comment]
Hey @xkrypt0nx,
Hope you are doing well. Let me discuss your concern with our development team and I will update you accordingly. Your patience would be highly appreciated. Thanks!
Regards,
Ayesha Aamir

**Comment by Repairdesk-QA** [comment]
Hey @xkrypt0nx,
I hope you are doing well. Your reported issue has been fixed. We have added a new trigger in the **Appointment Calendar Widget Configuration tab > Select image size for Manufacturers and Devices**, select **Large** option from the dropdown and hit SAVE changes. Refresh the widget & here you go the Manufacturers and Devices images will be displayed larger than before. I have also created a video for the same, kindly have a look at it.
https://drive.google.com/file/d/14BcnoZnY74TEHebqewbyefmj3dfBwvhV/view

And I hope that if there’s anything else I can do for you, you won’t hesitate to let me know. 
Thanks & Regards,
Ayesha Aamir
